app/controllers/parcels_controller.rb

```diff
@@ -28,7 +30,7 @@ def show
   private
 
   def parcel_params
-    params.require(:parcel).permit(:weight, :width, :depth, :height, :name)
+    params.require(:parcel).permit(:weight, :width, :depth, :height, :name, sender_info_attributes: [:email, :contact_name, :zip_code, :address, :city, :phone_number, :company_name, :other_info, :residential], recipient_info_attributes: [:email, :contact_name, :zip_code, :address, :city, :phone_number, :company_name, :other_info, :residential])
```

Comment:
Rozbij to plox na kilka linijek :)
https://github.com/bbatsov/ruby-style-guide#no-double-indent
(przy czym w ogóle polecam przejrzeć sobie te style-guide'y)